test/common/common.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Give a longer project name, e.g. Microsoft.Azure.IoT.Edge.Test.Framework.proj.

Reply:
As long as this library is scoped to the `test/` directory I don't see the need to give it a name outside that scope, and certainly not a derivation of our product code namespaces. I was following the pattern of our modules like temp sensor and load-gen, which have simple, independent namespaces that work for their intended scope.